services/download/src/main/java/app/coronawarn/server/services/download/FederationBatchProcessor.java

```diff
@@ -99,18 +99,18 @@ protected void saveFirstBatchInfoForDate(LocalDate date) throws FatalFederationG
       logger.info("Triggering download of first batch for date {}.", date);
       BatchDownloadResponse response = federationGatewayDownloadService.downloadBatch(date);
       batchInfoService.save(new FederationBatchInfo(response.getBatchTag(), date));
-    } catch (BatchDownloadException e) {
-      logger.error("Triggering download of first batch for date {} failed. Reason: {}.", date, e.getMessage());
     } catch (FatalFederationGatewayException e) {
       throw e;
     } catch (Exception e) {
-      logger.error("Triggering download of first batch for date {} failed.", date, e);
+      logger.error(
+          "Triggering download of first batch for date {} failed.",
```

Comment:
I personally like this in one line. What's your configured max line length? I think we have 120 chars in checkstyle.